Assets/Code/Interactables/Effects/Static Effects/Statuses/AttachedStatus.cs

```diff
@@ -0,0 +1,23 @@
+﻿public abstract class AttachedStatus : AttachedStaticEffect {
+    public StatusType statusType;
+
+    // How much status is currently applied
+    public int amount;
```

Comment:
probably want to standardize this as 'effectCount'

Reply:
and maybe have it at a higher level in the effect heirarchy?

Reply:
Agreed.